manticore/core/cpu/abstractcpu.py

```diff
@@ -518,7 +589,11 @@ def decode_instruction(self, pc):
                     if isinstance(c, Constant):
                         c = chr(c.value)
                     else:
+                        logger.info('Trying to execute instructions from invalid memory')
```

Comment:
doesn't the existing logger call here already convey basically this information?